# HG changeset patch
# User Torsten <mttl@mailbox.org>
# Date 1545947860 -3600
#      Thu Dec 27 22:57:40 2018 +0100
# Branch stable
# Node ID 046b2088843e58a16db3909e26504050d8cd1e4f
# Parent  a21f58c5e573087abd1f7502104f9d899068b607
* es_ES.ts: spanish translation by José Luis García Pallero (bug #55279)

diff --git a/libgui/languages/es_ES.ts b/libgui/languages/es_ES.ts
--- a/libgui/languages/es_ES.ts
+++ b/libgui/languages/es_ES.ts
@@ -101,17 +101,17 @@
     <message>
         <location line="+5"/>
         <source>Select All</source>
         <translation>Seleccionar todo</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Run Selection</source>
-        <translation type="unfinished">Ejecutar selección</translation>
+        <translation>Ejecutar selección</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Clear Window</source>
         <translation>Limpiar ventana</translation>
     </message>
     <message>
         <location filename="../qterminal/libqterminal/QTerminal.cc" line="+49"/>
@@ -253,18 +253,17 @@
     <message>
         <location line="+428"/>
         <source>color</source>
         <translation>color</translation>
     </message>
     <message>
         <location line="+22"/>
         <source>Horz alignment</source>
-        <translatorcomment>&quot;Alineación horizontal&quot; or &quot;Alineación hrz.&quot; (abbreviated form)</translatorcomment>
-        <translation type="unfinished">Alineación horizontal</translation>
+        <translation>Alineación horizontal</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>left</source>
         <translation>izquierda</translation>
     </message>
     <message>
         <location line="+5"/>
@@ -275,18 +274,17 @@
     <message>
         <location line="-24"/>
         <source>right</source>
         <translation>derecha</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Vert alignment</source>
-        <translatorcomment>&quot;Alineación vertical.&quot; or &quot;Alineación vert.&quot; (abbreviated form)</translatorcomment>
-        <translation type="unfinished">Alineación vertical</translation>
+        <translation>Alineación vertical</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>top</source>
         <translation>parte superior</translation>
     </message>
     <message>
         <location line="+10"/>
@@ -336,18 +334,18 @@
     <message>
         <location line="+13"/>
         <source>Box</source>
         <translation>Cuadro</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Background</source>
-        <translatorcomment>Segundo plano o fondo</translatorcomment>
-        <translation>Segundo plano</translation>
+        <translatorcomment>En este contexto es más adecuada la traducción &quot;fondo&quot; en lugar de &quot;segundo plano&quot;</translatorcomment>
+        <translation>Fondo</translation>
     </message>
     <message>
         <location line="+444"/>
         <source>Edge</source>
         <translatorcomment>Borde u orilla</translatorcomment>
         <translation>Borde</translation>
     </message>
     <message>
@@ -383,112 +381,115 @@
         <source>Octave Documentation</source>
         <translation>Documentación de Octave</translation>
     </message>
     <message>
         <location line="-624"/>
         <source>Could not copy help collection to temporary
 file. Search capabilities may be affected.
 %1</source>
-        <translation type="unfinished"></translation>
+        <translation>No se ha posido copiar la ayuda a un fichero temporal.
+Las capacidades de búsqueda pueden verse afectadas.
+%1</translation>
     </message>
     <message>
         <location line="+12"/>
         <source>Could not setup the data required for the
 documentation viewer. Only help texts in
 the Command Window will be available.</source>
-        <translation type="unfinished"></translation>
+        <translation>No se han podido establecer los datos requeridos para el visor de documentación.
+Sólo estará disponible la ayuda en la línea de comandos.</translation>
     </message>
     <message>
         <location line="+21"/>
         <source>Find:</source>
-        <translation type="unfinished"></translation>
+        <translation>Encontrar:</translation>
     </message>
     <message>
         <location line="+7"/>
         <location line="+1"/>
         <source>Search forward</source>
-        <translation type="unfinished"></translation>
+        <translation>Buscar hacia delante</translation>
     </message>
     <message>
         <location line="+5"/>
         <location line="+1"/>
         <source>Search backward</source>
-        <translation type="unfinished"></translation>
+        <translation>Buscar hacia atrás</translation>
     </message>
     <message>
         <location line="+74"/>
         <source>Function Index</source>
-        <translation type="unfinished"></translation>
+        <translation>Índice de funciones</translation>
     </message>
     <message>
         <location line="+98"/>
         <source>Go home</source>
-        <translation type="unfinished"></translation>
+        <translation>Ir al directorio de usuario</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Go back</source>
-        <translation type="unfinished"></translation>
+        <translation>Ir hacia atrás</translation>
     </message>
     <message>
         <location line="+14"/>
         <source>Go forward</source>
-        <translation type="unfinished"></translation>
+        <translation>Ir hacia delante</translation>
     </message>
     <message>
         <location line="+43"/>
         <source>Find</source>
-        <translation type="unfinished">Buscar</translation>
+        <translation>Buscar</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Zoom in</source>
-        <translation type="unfinished"></translation>
+        <translation>Ampliar</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Zoom out</source>
-        <translation type="unfinished"></translation>
+        <translation>Reducir</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Zoom original</source>
-        <translation type="unfinished"></translation>
+        <translation>Zoom original</translation>
     </message>
     <message>
         <location line="-206"/>
         <source>Contents</source>
         <translation>Contenido</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Enter text to search the indices</source>
-        <translation>Ingrese texto para buscar los índices</translation>
+        <translation>Introduzca texto para buscar en los índices</translation>
     </message>
     <message>
         <location line="+10"/>
         <location line="+39"/>
         <source>Search</source>
         <translation>Búsqueda</translation>
     </message>
     <message>
         <location line="+84"/>
         <source>Previous pages</source>
-        <translation type="unfinished"></translation>
+        <translation>Páginas anteriores</translation>
     </message>
     <message>
         <location line="+14"/>
         <source>Next pages</source>
-        <translation type="unfinished"></translation>
+        <translation>Páginas siguientes</translation>
     </message>
     <message>
         <location line="+383"/>
         <source>Unable to register help file %1.</source>
-        <translation type="unfinished"></translation>
+        <translation>Imposible mostrar el fichero de ayuda %1.</translation>
     </message>
 </context>
 <context>
     <name>octave::documentation_dock_widget</name>
     <message>
         <location filename="../src/documentation-dock-widget.cc" line="+38"/>
         <source>Documentation</source>
         <translation>Documentación</translation>
@@ -499,37 +500,37 @@ the Command Window will be available.</s
         <translation>Consulte la documentación para obtener ayuda.</translation>
     </message>
 </context>
 <context>
     <name>octave::dw_main_window</name>
     <message>
         <location filename="../src/dw-main-window.cc" line="+49"/>
         <source>&amp;Close</source>
-        <translation type="unfinished">C&amp;errar</translation>
+        <translation>C&amp;errar</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Close &amp;All</source>
-        <translation type="unfinished"></translation>
+        <translation>Cerrar &amp;todo</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Close &amp;Other</source>
-        <translation type="unfinished"></translation>
+        <translation>Cerrar los &amp;otros</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Switch to &amp;Left Widget</source>
-        <translation type="unfinished"></translation>
+        <translation>Calbiar al widget de la &amp;izquierda</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Switch to &amp;Right Widget</source>
-        <translation type="unfinished"></translation>
+        <translation>Calbiar al widget de la &amp;derecha</translation>
     </message>
 </context>
 <context>
     <name>octave::external_editor_interface</name>
     <message>
         <location filename="../src/external-editor-interface.cc" line="+63"/>
         <location line="+45"/>
         <source>Octave Editor</source>
@@ -541,17 +542,17 @@ the Command Window will be available.</s
 %1</source>
         <translation>No se puede abrir el editor de texto personalizado
 %1</translation>
     </message>
     <message>
         <location line="+45"/>
         <source>There is no custom editor configured yet.
 Do you want to open the preferences?</source>
-        <translation>No existe una configuración para editor personalizado aún.
+        <translation>Todavía no existe una configuración para editor personalizado.
 ¿Desea abrir las preferencias?</translation>
     </message>
 </context>
 <context>
     <name>octave::file_editor</name>
     <message>
         <location filename="../src/m-editor/file-editor.cc" line="+860"/>
         <location line="+28"/>
@@ -731,17 +732,17 @@ no puede ser abierto para escritura: %2.
         <location line="+7"/>
         <source>Duplicate Selection/Line</source>
         <translation>Duplicar selección/línea</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Transpose Line</source>
         <translatorcomment>also &quot;transponer línea&quot; or &quot;Intercambiar con línea anterior&quot;</translatorcomment>
-        <translation type="unfinished">Intercambiar línea</translation>
+        <translation>Intercambiar líneas</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>&amp;Show Completion List</source>
         <translation>&amp;Mostrar lista de comandos coincidentes</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -766,49 +767,49 @@ no puede ser abierto para escritura: %2.
     <message>
         <location line="+5"/>
         <source>&amp;Uncomment</source>
         <translation>&amp;Eliminar comentario</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Comment (Choosing String)</source>
-        <translation>Comentar (Seleccionando cadena de texto)</translation>
+        <translation>Comentar (seleccionando cadena de texto)</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>&amp;Indent Selection Rigidly</source>
         <translatorcomment>Would use it. If possible verify.</translatorcomment>
-        <translation type="unfinished">&amp;Sangría selección rigurosa</translation>
+        <translation>&amp;Sangría rigurosa de la selección</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&amp;Unindent Selection Rigidly</source>
         <translatorcomment>Would use it. If possible verify.</translatorcomment>
-        <translation type="unfinished">&amp;Eliminar Sangría selección rigurosa</translation>
+        <translation>&amp;Eliminar sangría rigurosa de la selección</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Indent Code</source>
         <translation>Aplicar sangría a código</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Convert Line Endings to &amp;Windows (CRLF)</source>
-        <translation type="unfinished"></translation>
+        <translation>Convertir caracteres de find e línea a &amp;Windows (CR+LF)</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Convert Line Endings to &amp;Unix (LF)</source>
-        <translation>Convertir los finales de línea a &amp;Unix (LF)</translation>
+        <translation>Convertir caracteres de find e línea a &amp;Unix (LF)</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Convert Line Endings to &amp;Mac (CR)</source>
-        <translation>Convertir los finales de línea a &amp;Mac (CR)</translation>
+        <translation>Convertir caracteres de find e línea a &amp;Mac (CR)</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Navi&amp;gation</source>
         <translation>&amp;Navegación</translation>
     </message>
     <message>
         <location line="+4"/>
@@ -873,17 +874,17 @@ no puede ser abierto para escritura: %2.
     <message>
         <location line="+6"/>
         <source>Show &amp;Whitespace Characters</source>
         <translation>Mostrar &amp;espacios en blanco</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Show Line &amp;Endings</source>
-        <translation>Mostrar &amp;fines de línea</translation>
+        <translation>Mostrar caracteres de &amp;fin de línea</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Show &amp;Indentation Guides</source>
         <translation>Mostrar guías de &amp;sangría</translation>
     </message>
     <message>
         <location line="+6"/>
@@ -973,17 +974,17 @@ no puede ser abierto para escritura: %2.
     <message>
         <location line="+5"/>
         <source>&amp;Documentation on Keyword</source>
         <translation>&amp;Documentación de palabras clave</translation>
     </message>
     <message>
         <location line="+25"/>
         <source>Recent Files</source>
-        <translation>Archivos Recientes</translation>
+        <translation>Archivos recientes</translation>
     </message>
     <message>
         <location line="+63"/>
         <source>Editor</source>
         <translation>Editor</translation>
     </message>
 </context>
 <context>
@@ -1026,17 +1027,18 @@ no puede ser abierto para escritura: %2.
         <location line="+36"/>
         <source>Octave Editor</source>
         <translation>Editor de Octave</translation>
     </message>
     <message>
         <location line="-1977"/>
         <source>Cannot add breakpoint to modified or unnamed file.
 Save and add breakpoint, or cancel?</source>
-        <translation type="unfinished"></translation>
+        <translation>No se puede añadir un punto de interrupción a un archivo modificado o sin nombre.
+¿Guardar y añadir el punto de interrupción o cancelar?</translation>
     </message>
     <message>
         <location line="+833"/>
         <source>Goto line</source>
         <translation>Ir a la línea</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -1048,17 +1050,18 @@ Save and add breakpoint, or cancel?</sou
         <source>Comment selected text</source>
         <translation>Comentar texto seleccionado</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Comment string to use:
 </source>
         <translatorcomment>Not sure here.</translatorcomment>
-        <translation>Comentar cadena de texto a utilizar:</translation>
+        <translation>Comentar cadena de texto a utilizar:
+</translation>
     </message>
     <message>
         <location line="+135"/>
         <location line="+60"/>
         <source>&lt;unnamed&gt;</source>
         <translation>&lt;sin nombre&gt;</translation>
     </message>
     <message>
@@ -1081,48 +1084,56 @@ está a punto de ser cerrado, pero ha sido modificado.  %2</translation>
     </message>
     <message>
         <location line="+126"/>
         <source>There were problems reading the file
 %1
 with the selected encoding %2.
 
 Modifying and saving the file might cause data loss!</source>
-        <translation type="unfinished"></translation>
+        <translation>Hubo problemas en la lectura del archivo
+%1
+con la codificación seleccionada %2.
+
+¡La modificación y guardado del archivo podría significar una pérdida de datos!</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>&amp;Edit anyway</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Editar de todos modos</translation>
     </message>
     <message>
         <location line="+1"/>
         <location line="+57"/>
         <source>Chan&amp;ge encoding</source>
-        <translation type="unfinished"></translation>
+        <translation>Cambi&amp;ar codificación</translation>
     </message>
     <message>
         <location line="-56"/>
         <location line="+49"/>
         <source>&amp;Close</source>
-        <translation type="unfinished">C&amp;errar</translation>
+        <translation>C&amp;errar</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>Select new default encoding</source>
-        <translation type="unfinished"></translation>
+        <translation>Seleccione una nueva codificación por defecto</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Please select a new encoding
 for reloading the current file.
 
 This does not change the default encoding.
 </source>
-        <translation type="unfinished"></translation>
+        <translation>Por favor, seleccione una nueva codificación
+para la recarga del archivo actual.
+
+Esto no cambiará la codificación predeterminada.
+</translation>
     </message>
     <message>
         <location line="+212"/>
         <source>Debug or Save</source>
         <translation>Depurar programa o guardar</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -1139,26 +1150,33 @@ Quit debugging and save?</source>
 %2.</translation>
     </message>
     <message>
         <location line="+182"/>
         <source>The current encoding %1
 can not be applied.
 
 Please select another one!</source>
-        <translation type="unfinished"></translation>
+        <translation>La codificación actual %1
+no se puede aplicar.
+
+¡Por favor, seleccione otra!</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>The current editor contents can not be encoded
 with the selected encoding %1.
 Using it would result in data loss!
 
 Please select another one!</source>
-        <translation type="unfinished"></translation>
+        <translation>El contenido actual del editor no puede ser codificado
+con la codificación seleccionada %1.
+¡Su utilización podría ocasionar una pérdida de datos!
+
+¡Por favor, seleccione otra codificación!</translation>
     </message>
     <message>
         <location line="-115"/>
         <source>Octave Files (*.m)</source>
         <translation>Archivos de Octave (*.m)</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -1211,17 +1229,17 @@ ha sido eliminado o renombrado. ¿Desea salvarlo ahora?%2</translation>
     <message>
         <location filename="../src/files-dock-widget.cc" line="+73"/>
         <source>File Browser</source>
         <translation>Explorador de archivos</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Browse your files</source>
-        <translation>Explorar archivos</translation>
+        <translation>Explorar sus archivos</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>File size</source>
         <translation>Tamaño del archivo</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -1305,17 +1323,17 @@ ha sido eliminado o renombrado. ¿Desea salvarlo ahora?%2</translation>
         <location line="-381"/>
         <location line="+384"/>
         <source>New Directory...</source>
         <translation>Nuevo directorio...</translation>
     </message>
     <message>
         <location line="-329"/>
         <source>Double-click to open file/folder, right click for alternatives</source>
-        <translation type="unfinished"></translation>
+        <translation>Doble clic para abrir el archivo o la carpeta; clic en el botón derecho para ver las alternativas</translation>
     </message>
     <message>
         <location line="+282"/>
         <source>Open</source>
         <translation>Abrir</translation>
     </message>
     <message>
         <location line="+5"/>
@@ -1462,17 +1480,17 @@ ha sido eliminado o renombrado. ¿Desea salvarlo ahora?%2</translation>
         <location line="+7"/>
         <source>&lt;html&gt;&lt;body&gt;
 &lt;p&gt;We hope you find Octave to be a useful tool.&lt;/p&gt;
 &lt;p&gt;If you encounter problems, there are a number of ways to get help, including commercial support options, a mailing list, a wiki, and other community-based support channels.
 You can find more information about each of these by visiting &lt;a href=&quot;https://octave.org/support.html&quot;&gt;https://octave.org/support.html&lt;/a&gt; (opens in external browser).&lt;/p&gt;
 &lt;/body&gt;&lt;/html&gt;</source>
         <translation>&lt;html&gt;&lt;body&gt;
 &lt;p&gt;Esperamos que Octave le sea útil.&lt;/p&gt;
-&lt;p&gt;Si se encuentra con problemas, existen varias formas de obtener ayuda, las cuales incluyen soporte comercial, listas de correo, un wiki y otros canales de comunicación sostenidos por la comunidad.
+&lt;p&gt;Si se encuentra con problemas, existen varias formas de obtener ayuda, las cuales incluyen soporte comercial, listas de correo, un wiki y otros canales de comunicación mantenidos por la comunidad.
 Puede encontrar más información visitando &lt;a href=&quot;https://octave.org/support.html&quot;&gt;https://octave.org/support.html&lt;/a&gt; (se abre en una ventana externa del navegador).&lt;/p&gt;
 &lt;/body&gt;&lt;/html&gt;</translation>
     </message>
     <message>
         <location line="+21"/>
         <source>&lt;html&gt;&lt;head&gt;
 &lt;style&gt;
 a:link { text-decoration: underline; color: #0000ff; }
@@ -1491,20 +1509,17 @@ a:link { text-decoration: underline; col
 &lt;/style&gt;
 &lt;head/&gt;&lt;body&gt;
 &lt;p&gt;Para más información acerca de Octave:&lt;/p&gt;
 &lt;ul&gt;
 &lt;li&gt;Visite &lt;a href=&quot;https://octave.org&quot;&gt;https://octave.org&lt;/a&gt; (se abre en una ventana externa del navegador)&lt;/li&gt;
 &lt;li&gt;Obtenga la documentación en línea en formato &lt;a href=&quot;https://www.gnu.org/software/octave/doc/interpreter/index.html&quot;&gt;html&lt;/a&gt; o como &lt;a href=&quot;https://www.gnu.org/software/octave/octave.pdf&quot;&gt;documento&lt;/span&gt;&lt;/a&gt; pdf (se abre en una ventana externa del navegador)&lt;/li&gt;
 &lt;li&gt;Abra el navegador de documentación de la GUI de Octave desde el menú de ayuda&lt;/li&gt;
 &lt;/ul&gt;
-&lt;/body&gt;&lt;/html&gt;
-
-
-</translation>
+&lt;/body&gt;&lt;/html&gt;</translation>
     </message>
 </context>
 <context>
     <name>octave::find_dialog</name>
     <message>
         <location filename="../src/m-editor/find-dialog.cc" line="+87"/>
         <source>Find and Replace</source>
         <translation>Buscar y reemplazar</translation>
@@ -1593,17 +1608,17 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+209"/>
         <source>Replace Result</source>
         <translation>Reemplazar resultado</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>%1 items replaced</source>
-        <translation>%1 instancias reemplazadas</translation>
+        <translation>%1 instancia(s) reemplazada(s)</translation>
     </message>
     <message>
         <location line="+14"/>
         <source>Find Result</source>
         <translation>Resultado de la búsqueda</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -1682,17 +1697,17 @@ a:link { text-decoration: underline; col
         <location line="+2"/>
         <source>Contains text:</source>
         <translation>Que contenga el texto:</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Enter the file content search expression</source>
         <translatorcomment>Please double check this!</translatorcomment>
-        <translation type="unfinished">Introducir expresión de búsuqeda en el contenido del archivo</translation>
+        <translation>Introducir expresión de búsqueda en el contenido del archivo</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Text to match</source>
         <translation>Texto a coincidir</translation>
     </message>
     <message>
         <location line="+4"/>
@@ -1747,17 +1762,17 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+103"/>
         <source>Searching...</source>
         <translation>Buscando...</translation>
     </message>
     <message>
         <location line="+22"/>
         <source>%1 match (es)</source>
-        <translation>%1 coincidencia (s)</translation>
+        <translation>%1 coincidencia(s)</translation>
     </message>
     <message>
         <location line="+14"/>
         <source>Set search directory</source>
         <translation>Seleccionar el directorio de búsqueda</translation>
     </message>
 </context>
 <context>
@@ -1788,32 +1803,32 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+1"/>
         <source>Evaluate</source>
         <translation>Evaluar</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Create script</source>
-        <translation>Crear un guión (script)</translation>
+        <translation>Crear un guion (script)</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Hide filter</source>
         <translation>Ocultar filtro</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Show filter</source>
         <translation>Mostrar filtro</translation>
     </message>
     <message>
         <location line="+121"/>
         <source>Double-click a command to transfer it to the Command Window.</source>
-        <translation type="unfinished"></translation>
+        <translation>Haga doble clic en un comando para introducirlo en la ventana de comandos.</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Enter text to filter the command history</source>
         <translation>Introduzca texto para filtrar el historial de comandos</translation>
     </message>
     <message>
         <location line="+10"/>
@@ -1933,23 +1948,25 @@ Pulse &apos;Siguiente&apos; para crear un archivo de configuración e iniciar Octave.&lt;/p&gt;
         <location line="+37"/>
         <source>New Function</source>
         <translation>Nueva función</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>New function name:
 </source>
-        <translation>Nombre para la nueva función:</translation>
+        <translation>Nombre para la nueva función:
+</translation>
     </message>
     <message>
         <location line="+44"/>
         <source>%1 is a built-in, compiled or inline
 function and can not be edited.</source>
-        <translation type="unfinished"></translation>
+        <translation>%1 es una función integrada (built-in), compilada o inline,
+por lo que no puede ser editada.</translation>
     </message>
     <message>
         <location line="+40"/>
         <source>Can not find function %1</source>
         <translation>No se puede encontrar la función %1</translation>
     </message>
     <message>
         <location line="+9"/>
@@ -1989,17 +2006,17 @@ function and can not be edited.</source>
     <message>
         <location line="+18"/>
         <source>New</source>
         <translation>Nuevo</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>New Script</source>
-        <translation>Nuevo guión (script)</translation>
+        <translation>Nuevo guion (script)</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>New Function...</source>
         <translation>Nueva función...</translation>
     </message>
     <message>
         <location line="+4"/>
@@ -2263,22 +2280,22 @@ when you have a connection to the web (l
 &lt;p&gt;
 &lt;small&gt;&lt;em&gt;&amp;mdash; The Octave Developers, </source>
         <translation>&lt;html&gt;
 &lt;body&gt;
 &lt;p&gt;
 Las fuentes de noticias de la comunidad Octave parece que no están disponibles.
 &lt;/p&gt;
 &lt;p&gt;
-Para ver las últimas novedades, visite
+Para ver las últimas novedades visite
 &lt;a href=&quot;https://octave.org/community-news.html&quot;&gt;https://octave.org/community-news.html&lt;/a&gt;
 cuando tenga conexión a la web (el enlace se abre en un navegador externo).
 &lt;/p&gt;
 &lt;p&gt;
-&lt;small&gt;&lt;em&gt;&amp;mdash; Los desarrolladores de Octave,</translation>
+&lt;small&gt;&lt;em&gt;&amp;mdash; Los desarrolladores de Octave, </translation>
     </message>
     <message>
         <location line="+18"/>
         <source>&lt;html&gt;
 &lt;body&gt;
 &lt;p&gt;
 Connecting to the web to display the latest Octave Community news has been disabled.
 &lt;/p&gt;
@@ -2291,24 +2308,23 @@ or enable web connections for news in Oc
 &lt;p&gt;
 &lt;small&gt;&lt;em&gt;&amp;mdash; The Octave Developers, </source>
         <translation>&lt;html&gt;
 &lt;body&gt;
 &lt;p&gt;
 La conexión a la web para mostrar las últimas novedades sobre Octave ha sido desactivada.
 &lt;/p&gt;
 &lt;p&gt;
-Para ver las últimas novedades, vaya a
+Para ver las últimas novedades vaya a
 &lt;a href=&quot;https://octave.org/community-news.html&quot;&gt;https://octave.org/community-news.html&lt;/a&gt;
 cuando tenga conexión a la web (el enlace se abre en un navegador externo)
 o actívela para ver las novedades en el diálogo de configuración de red de Octave
 &lt;/p&gt;
 &lt;p&gt;
-&lt;small&gt;&lt;em&gt;&amp;mdash; Los desarrolladores de Octave,
-</translation>
+&lt;small&gt;&lt;em&gt;&amp;mdash; Los desarrolladores de Octave, </translation>
     </message>
 </context>
 <context>
     <name>octave::octave_dock_widget</name>
     <message>
         <location filename="../src/octave-dock-widget.cc" line="+138"/>
         <source>Hide widget</source>
         <translation>Ocultar widget</translation>
@@ -2379,48 +2395,45 @@ no existe. ¿Desea crearlo?</translation>
     <message>
         <location line="+2"/>
         <source>Octave Editor</source>
         <translation>Editor de Octave</translation>
     </message>
     <message>
         <location line="+245"/>
         <source>The file %1 does not exist in the load path.  To run or debug the function you are editing, you must either change to the directory %2 or add that directory to the load path.</source>
-        <translatorcomment>dirección o ruta de carga.</translatorcomment>
-        <translation type="unfinished">El archivo %1 no existe en las ruta de carga (dirección). Para ejecutar o depurar la función que está editando tiene que cambiar al directorio %2 o agregarlo a las rutas de carga (path).</translation>
+        <translation>El archivo %1 no existe en las rutas de carga (load path). Para ejecutar o depurar la función que está editando tiene que cambiar al directorio %2 o agregarlo a las rutas de carga (load path).</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>The file %1 is shadowed by a file with the same name in the load path. To run or debug the function you are editing, change to the directory %2.</source>
-        <translatorcomment>dirección o ruta de carga.</translatorcomment>
-        <translation type="unfinished">Existe un archivo con el mismo nombre que %1 en la ruta de carga (dirección). Para ejecutar o depurar la función que se está editando cambie al directorio %2.</translation>
+        <translation>Existe un archivo con el mismo nombre que %1 en la ruta de carga (load path). Para ejecutar o depurar la función que se está editando cambie al directorio %2.</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Change Directory or Add Directory to Load Path</source>
-        <translatorcomment>dirección o ruta de carga.</translatorcomment>
-        <translation type="unfinished">Cambiar directorio o agregar directorio a la ruta de carga (dirección)</translation>
+        <translation>Cambiar directorio o agregar directorio a la ruta de carga (load path)</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Change Directory</source>
         <translation>&amp;Cambiar directorio</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>&amp;Add Directory to Load Path</source>
-        <translation>&amp;Agregar directorio a ruta de carga</translation>
+        <translation>&amp;Agregar directorio a ruta de carga (load path)</translation>
     </message>
 </context>
 <context>
     <name>octave::octave_txt_lexer</name>
     <message>
         <location filename="../src/m-editor/octave-txt-lexer.cc" line="+40"/>
         <source>Default</source>
-        <translation>Valor predeterminado/por defecto</translation>
+        <translation>Valor predeterminado</translation>
     </message>
 </context>
 <context>
     <name>octave::resource_manager</name>
     <message>
         <location filename="../src/resource-manager.cc" line="+302"/>
         <source>The settings file
 %1
@@ -2430,77 +2443,77 @@ Make sure you have read and write permis
 
 Octave GUI must be closed now.</source>
         <translation>El archivo de configuración
 %1
 no existe y no puede ser creado.
 Asegúrese de que tiene permisos de lectura y escritura en
 %2
 
-La interfaz gráfica de Octave se cerrará ahora.</translation>
+La interfaz gráfica de Octave se cerrará.</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Octave Critical Error</source>
         <translation>Error crítico de Octave</translation>
     </message>
 </context>
 <context>
     <name>octave::settings_dialog</name>
     <message>
         <location filename="../src/settings-dialog.cc" line="-573"/>
         <source>Octave Preferences</source>
-        <translation type="unfinished"></translation>
+        <translation>Preferencias de Octave</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Unable to save preferences.  Missing preferences file or unknown directory.</source>
-        <translation type="unfinished"></translation>
+        <translation>Imposible guardar las preferencias. No hay archivo de preferencias o directorio desconocido.</translation>
     </message>
     <message>
         <location line="+15"/>
         <location line="+4"/>
         <location line="+700"/>
         <source>System setting</source>
-        <translation type="unfinished">Configuración del sistema</translation>
+        <translation>Configuración del sistema</translation>
     </message>
     <message>
         <location line="-448"/>
         <source>IBeam Cursor</source>
-        <translation type="unfinished">Cursor IBeam</translation>
+        <translation>Cursor IBeam</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Block Cursor</source>
-        <translation type="unfinished">Cursor de bloque</translation>
+        <translation>Cursor de bloque</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Underline Cursor</source>
-        <translation type="unfinished">Cursor de subrayado</translation>
+        <translation>Cursor de subrayado</translation>
     </message>
     <message>
         <location line="+164"/>
         <source>Set Octave Startup Directory</source>
-        <translation type="unfinished">Ajuste del directorio de inicio de Octave</translation>
+        <translation>Ajuste del directorio de inicio de Octave</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Set File Browser Startup Directory</source>
-        <translation type="unfinished">Ajuste del directorio de inicio del explorador de archivos</translation>
+        <translation>Ajuste del directorio de inicio del explorador de archivos</translation>
     </message>
     <message>
         <location line="+503"/>
         <source>Enable attribute colors</source>
-        <translation type="unfinished"></translation>
+        <translation>Activar atributor de color</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Hide tools tips</source>
-        <translation type="unfinished"></translation>
+        <translation>Ocultar ayuda emergente</translation>
     </message>
 </context>
 <context>
     <name>octave::setup_community_news</name>
     <message>
         <location filename="../src/welcome-wizard.cc" line="+42"/>
         <source>Community News</source>
         <translation>Noticias de la comunidad</translation>
@@ -2566,17 +2579,17 @@ a:link { text-decoration: underline; col
   &quot;%1&quot;
 is already used for the action
   &quot;%2&quot;.
 Do you want to use the shortcut anyhow removing it from the previous action?</source>
         <translation>El acceso directo seleccionado
   &quot;%1&quot;
 ya se utiliza para la acción
   &quot;%2&quot;.
-¿Desea utilizar el acceso directo de cualquier modo suprimiendo la acción previamente asignada?</translation>
+¿Desea utilizarlo de cualquier modo suprimiendo la acción previamente asignada?</translation>
     </message>
     <message>
         <location line="+102"/>
         <source>New File</source>
         <translation>Nuevo archivo</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -2927,17 +2940,17 @@ ya se utiliza para la acción
         <location line="+2"/>
         <source>Duplicate Selection/Line</source>
         <translation>Duplicar selección/línea</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Transpose Line</source>
         <translatorcomment>also &quot;transponer línea&quot; or &quot;Intercambiar con línea anterior&quot;</translatorcomment>
-        <translation type="unfinished">Intercambiar línea</translation>
+        <translation>Intercambiar líneas</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Show Completion List</source>
         <translation>Mostrar lista de comandos coincidentes</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -2947,59 +2960,59 @@ ya se utiliza para la acción
     <message>
         <location line="+2"/>
         <source>Uncomment Selection</source>
         <translation>Descomentar la selección</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Comment Selection (Choosing String)</source>
-        <translation>Comentar selección (Seleccionando cadena de texto)</translation>
+        <translation>Comentar selección (seleccionando cadena de texto)</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Uppercase Selection</source>
         <translation>Convertir la selección a mayúsculas</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Lowercase Selection</source>
         <translation>Convertir la selección a minúsculas</translation>
     </message>
     <message>
         <location line="+4"/>
         <location line="+5"/>
         <source>Indent Selection Rigidly</source>
-        <translation>Sangría selección rigurosa</translation>
+        <translation>Sangría rigurosa de la selección</translation>
     </message>
     <message>
         <location line="-3"/>
         <location line="+5"/>
         <source>Unindent Selection Rigidly</source>
-        <translation>Eliminar sangría selección rigurosa</translation>
+        <translation>Eliminar sangría rigurosa de la selección</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Indent Code</source>
         <translation>Aplicar sangría a código</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Convert Line Endings to Windows</source>
-        <translation>Convertir fines de líneas a Windows</translation>
+        <translation>Convertir caracteres de fin de línea a Windows</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Convert Line Endings to Unix</source>
-        <translation>Convertir fines de líneas a Unix</translation>
+        <translation>Convertir caracteres de fin de línea a Unix</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Convert Line Endings to Mac</source>
-        <translation>Convertir fines de líneas a Mac</translation>
+        <translation>Convertir caracteres de fin de línea a Mac</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Goto Line</source>
         <translation>Ir a la línea</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -3074,53 +3087,53 @@ ya se utiliza para la acción
     <message>
         <location line="+2"/>
         <source>Show Horizontal Scrollbar</source>
         <translation>Mostrar barra de desplazamiento horizontal</translation>
     </message>
     <message>
         <location line="+154"/>
         <source>Documentation Viewer</source>
-        <translation type="unfinished"></translation>
+        <translation>Visor de documentación</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Browser</source>
-        <translation type="unfinished"></translation>
+        <translation>Explorador</translation>
     </message>
     <message>
         <location line="+79"/>
         <source>Import shortcuts from file...</source>
-        <translation type="unfinished"></translation>
+        <translation>Importar accesos directos de archivo...</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Export shortcuts to file...</source>
-        <translation type="unfinished"></translation>
+        <translation>Exportar accesos directos a archivo...</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>Failed to open %1 as Octave shortcut file</source>
-        <translation type="unfinished"></translation>
+        <translation>Fallo al abrir %1 como fichero de accesos directos de Octave</translation>
     </message>
     <message>
         <location line="-367"/>
         <source>Zoom In</source>
         <translation>Ampliar</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Zoom Out</source>
         <translation>Reducir</translation>
     </message>
     <message>
         <location line="+2"/>
         <location line="+3"/>
         <source>Zoom Normal</source>
-        <translation>Normal</translation>
+        <translation>Zoom normal</translation>
     </message>
     <message>
         <location line="+112"/>
         <source>Toggle Breakpoint</source>
         <translation>Alternar puntos de interrupción</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -3155,27 +3168,27 @@ ya se utiliza para la acción
     <message>
         <location line="+2"/>
         <source>Document on Keyword</source>
         <translation>Documentación de palabras clave</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Go to Homepage</source>
-        <translation type="unfinished"></translation>
+        <translation>Ir a la página principal</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Go Back one Page</source>
-        <translation type="unfinished"></translation>
+        <translation>Una página hacia atrás</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Go Forward one Page</source>
-        <translation type="unfinished"></translation>
+        <translation>Una página hacia delante</translation>
     </message>
     <message>
         <location line="+68"/>
         <source>Global</source>
         <translation>Global</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -3214,22 +3227,22 @@ ya se utiliza para la acción
     <message>
         <location line="+2"/>
         <source>Tab Handling in Dock Widgets</source>
         <translation>Manejo de pestañas en widgets acoplados</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Find &amp; Replace in Dock Widgets</source>
-        <translation type="unfinished"></translation>
+        <translation>Buscar y reemplazar en widgets acoplados</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Zooming in Editor and Documentation</source>
-        <translation type="unfinished"></translation>
+        <translation>Zoom en el editor y documentación</translation>
     </message>
     <message>
         <location line="+21"/>
         <source>View Menu</source>
         <translation>Menú ver</translation>
     </message>
     <message>
         <location line="+4"/>
@@ -3308,17 +3321,17 @@ Would you like to save the current short
     <message>
         <location line="+13"/>
         <source>Undock widget</source>
         <translation>Desacoplar widget</translation>
     </message>
     <message>
         <location line="+21"/>
         <source>Restore geometry</source>
-        <translation>Reestablecer geometría</translation>
+        <translation>Restablecer geometría</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Redock</source>
         <translation>Reacoplar</translation>
     </message>
 </context>
 <context>
@@ -3331,37 +3344,38 @@ Would you like to save the current short
     <message>
         <location line="+1"/>
         <source>Edit variables.</source>
         <translation>Editar variables.</translation>
     </message>
     <message>
         <location line="+281"/>
         <source>Foreground</source>
-        <translation type="unfinished"></translation>
+        <translation>Primer plano</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Background</source>
-        <translation type="unfinished">Segundo plano</translation>
+        <translation>Segundo plano</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Selected Foreground</source>
-        <translation type="unfinished"></translation>
+        <translation>Primer plano seleccionado</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Selected Background</source>
-        <translation type="unfinished"></translation>
+        <translation>Segundo plano seleccionado</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Alternate Background</source>
-        <translation type="unfinished"></translation>
+        <translatorcomment>No estoy seguro de esta traducción</translatorcomment>
+        <translation type="unfinished">Alternan segundo plano</translation>
     </message>
     <message>
         <location line="+273"/>
         <source>Variable Editor Toolbar</source>
         <translation>Barra de herramientas de editor de variables</translation>
     </message>
     <message>
         <location line="+4"/>
@@ -3403,37 +3417,37 @@ Would you like to save the current short
         <source>Paste clipboard into variable data</source>
         <translation>Pegar portapapeles a variable de datos</translation>
     </message>
     <message>
         <location line="+9"/>
         <location line="+5"/>
         <location line="+7"/>
         <source>Plot</source>
-        <translation>Graficar</translation>
+        <translation>Dibujar</translation>
     </message>
     <message>
         <location line="-11"/>
         <source>Plot Selected Data</source>
-        <translation>Graficar datos seleccionados</translation>
+        <translation>Dibujar datos seleccionados</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Plot selected data</source>
-        <translation>Graficar datos seleccionados</translation>
+        <translation>Dibujar datos seleccionados</translation>
     </message>
     <message>
         <location line="+21"/>
         <source>Up</source>
         <translation>Hacia arriba</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Go one level up in variable hierarchy</source>
-        <translation>Ir un nivel hacia arriba en jerarquia de variable</translation>
+        <translation>Ir un nivel hacia arriba en jerarquía de variables</translation>
     </message>
 </context>
 <context>
     <name>octave::variable_editor_stack</name>
     <message>
         <location line="-1212"/>
         <source>Save Variable %1 As</source>
         <translation>Guardar variable %1 como</translation>
@@ -3510,17 +3524,17 @@ Would you like to save the current short
     <message>
         <location line="+1"/>
         <source>Attribute</source>
         <translation>Atributo</translation>
     </message>
     <message>
         <location line="+108"/>
         <source>Right click to copy, rename, or display</source>
-        <translation>Clic derecho para copiar, renombrar o mostrar</translation>
+        <translation>Clic con el botón derecho para copiar, renombrar o mostrar</translation>
     </message>
     <message>
         <location line="+37"/>
         <location line="+2"/>
         <source>complex</source>
         <translation>complejo</translation>
     </message>
 </context>
@@ -3653,17 +3667,17 @@ Would you like to save the current short
     <message>
         <location line="+190"/>
         <source>Custom style</source>
         <translation>Estilo personalizado</translation>
     </message>
     <message>
         <location line="+133"/>
         <source>These preferences are applied after any .octaverc startup files.</source>
-        <translation>Estas configuraciones se aplicarán después de la ejecución de los archivos de inicio .octaverc.</translation>
+        <translation>Estas configuraciones se aplicarán después de la carga del archivo de inicio .octaverc.</translation>
     </message>
     <message>
         <location line="+353"/>
         <source>Editor</source>
         <translation>Editor</translation>
     </message>
     <message>
         <location line="+314"/>
@@ -3673,22 +3687,22 @@ Would you like to save the current short
     <message>
         <location line="+49"/>
         <source>Show EOL characters</source>
         <translation>Mostrar caracteres de fin de línea</translation>
     </message>
     <message>
         <location line="+309"/>
         <source>Wrap long lines at current window border</source>
-        <translation type="unfinished"></translation>
+        <translation>Ajustar las líneas demasiado largas al tamaño de la ventana</translation>
     </message>
     <message>
         <location line="+501"/>
         <source>Default EOL mode</source>
-        <translation>Identificador de fin de línea por defecto</translation>
+        <translation>Carácter de fin de línea por defecto</translation>
     </message>
     <message>
         <location line="-446"/>
         <source>Indent width</source>
         <translation>Ancho de sangría</translation>
     </message>
     <message>
         <location line="+7"/>
@@ -3864,17 +3878,17 @@ Would you like to save the current short
         <location line="+1705"/>
         <source>Browse</source>
         <translation>Explorar</translation>
     </message>
     <message>
         <location line="-1355"/>
         <source>Tab width min.</source>
         <translatorcomment>Ancho del tabulador</translatorcomment>
-        <translation>Extensión mínima del tabulador.</translation>
+        <translation>Extensión mínima del tabulador</translation>
     </message>
     <message>
         <location line="+70"/>
         <source>max.</source>
         <translation>máx.</translation>
     </message>
     <message>
         <location line="+422"/>
@@ -3885,17 +3899,17 @@ Would you like to save the current short
     <message>
         <location line="-463"/>
         <source>Enable Code Folding</source>
         <translation>Activar plegado de código</translation>
     </message>
     <message>
         <location line="+987"/>
         <source>Windows (CRLF)</source>
-        <translation>Windows (CRLF)</translation>
+        <translation>Windows (CR+LF)</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Mac (CR)</source>
         <translation>Mac (CR)</translation>
     </message>
     <message>
         <location line="+5"/>
@@ -3905,92 +3919,92 @@ Would you like to save the current short
     <message>
         <location line="-962"/>
         <source>Show horizontal scroll bar</source>
         <translation>Mostrar barra de desplazamiento horizontal</translation>
     </message>
     <message>
         <location line="-1037"/>
         <source>Preferences</source>
-        <translation type="unfinished">Preferencias</translation>
+        <translation>Preferencias</translation>
     </message>
     <message>
         <location line="+57"/>
         <source>(requires restart)</source>
-        <translation type="unfinished"></translation>
+        <translation>(requiere reinicio)</translation>
     </message>
     <message>
         <location line="+32"/>
         <source>Use native file dialogs</source>
-        <translation type="unfinished"></translation>
+        <translation>Usar selector de archivos nativo</translation>
     </message>
     <message>
         <location line="+53"/>
         <source>Use system icon theme if available (requires restart)</source>
-        <translation type="unfinished"></translation>
+        <translation>Usar tema de iconos del sistema, si está disponible (requiere reinicio)</translation>
     </message>
     <message>
         <location line="+70"/>
         <source>Toolbar Icons</source>
-        <translation type="unfinished"></translation>
+        <translation>Iconos de la barra de herramientas</translation>
     </message>
     <message>
         <location line="+186"/>
         <source>Language</source>
-        <translation type="unfinished"></translation>
+        <translation>Lenguaje</translation>
     </message>
     <message>
         <location line="+27"/>
         <source>Style</source>
-        <translation type="unfinished"></translation>
+        <translation>Estilo</translation>
     </message>
     <message>
         <location line="+27"/>
         <source>Initial working directory of Octave interpreter</source>
-        <translation type="unfinished"></translation>
+        <translation>Directorio de trabajo de inicio del intérprete de Octave</translation>
     </message>
     <message>
         <location line="+18"/>
         <source>Restore last working directory of previous session</source>
-        <translation type="unfinished"></translation>
+        <translation>Restaurar el directorio de trabajo de la sesión anterior</translation>
     </message>
     <message>
         <location line="+45"/>
         <source>Command</source>
-        <translation type="unfinished"></translation>
+        <translation>Comandos</translation>
     </message>
     <message>
         <location line="+208"/>
         <source>Set focus to Command Window when running a command from within another widget</source>
-        <translation type="unfinished"></translation>
+        <translation>Establecer el foco en la ventana de comandos cuando se ejecute un comando en otro widget</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Print debug location in Command Window in addition to the marker in the editor</source>
-        <translation type="unfinished"></translation>
+        <translation>Mostrar el punto de depuración en la ventana de comandos además de en el editor</translation>
     </message>
     <message>
         <location line="+16"/>
         <source>Command Window Colors</source>
-        <translation type="unfinished"></translation>
+        <translation>Colores de la ventana de comandos</translation>
     </message>
     <message>
         <location line="+172"/>
         <source>Show complete path in title</source>
         <translation>Mostrar ruta completa en el título</translation>
     </message>
     <message>
         <location line="+100"/>
         <source>Show tool bar</source>
         <translation>Mostrar barra de herramientas</translation>
     </message>
     <message>
         <location line="+31"/>
         <source>Number size as difference to editor font</source>
-        <translation>Tamaño del número como diferencia a la fuente del editor </translation>
+        <translation>Tamaño del número como diferencia con la fuente del editor</translation>
     </message>
     <message>
         <location line="+124"/>
         <source>Highlight all occurrences of a word selected by a double click</source>
         <translation>Resaltar todas las apariciones de palabra seleccionada con doble clic</translation>
     </message>
     <message>
         <location line="+65"/>
@@ -4010,17 +4024,16 @@ Would you like to save the current short
     <message>
         <location line="+31"/>
         <source>Long lines</source>
         <translation>Líneas largas</translation>
     </message>
     <message>
         <location line="+25"/>
         <source>Break long lines at line length</source>
-        <translatorcomment>Break=&quot;romper&quot; or &quot;partir&quot;. For this example not 100% sure.</translatorcomment>
         <translation>Partir líneas largas a partir de longitud de línea</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>Break lines only in comments</source>
         <translation>Partir líneas solo en comentarios</translation>
     </message>
     <message>
@@ -4036,43 +4049,42 @@ Would you like to save the current short
     <message>
         <location line="+10"/>
         <source>Line</source>
         <translation>Línea</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Background</source>
-        <translatorcomment>Segundo plano o fondo</translatorcomment>
-        <translation>Segundo plano</translation>
+        <translation>Fondo</translation>
     </message>
     <message>
         <location line="+53"/>
         <source>Indentation</source>
         <translation>Sangrado</translation>
     </message>
     <message>
         <location line="+127"/>
         <source>Indentation uses tabs</source>
         <translation>Sangrado con tabulador</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>Auto insert after &quot;if&quot; etc.</source>
-        <translation>Inserción automática después de &quot;if&quot; etc.</translation>
+        <translation>Inserción automática después de &quot;if&quot;, etc.</translation>
     </message>
     <message>
         <location line="+20"/>
         <source>Nothing</source>
-        <translation>Nada</translation>
+        <translation>nada</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&quot;endif&quot; etc.</source>
-        <translation>&quot;endif&quot; etc.</translation>
+        <translation>&quot;endif&quot;, etc.</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&quot;end&quot;</source>
         <translation>&quot;end&quot;</translation>
     </message>
     <message>
         <location line="+29"/>
@@ -4092,22 +4104,23 @@ Would you like to save the current short
     <message>
         <location line="+64"/>
         <source>Show completion list automatically</source>
         <translation>Mostrar lista de comandos coincidentes automáticamente</translation>
     </message>
     <message>
         <location line="+34"/>
         <source>Debugging</source>
-        <translation type="unfinished"></translation>
+        <translation>Depurado</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Always show debug breakpoints and pointers (opens related file if closed)</source>
-        <translation type="unfinished"></translation>
+        <translatorcomment>Traducción no muy bien hecha</translatorcomment>
+        <translation type="unfinished">Mostrar siempre puntos de depurado y punteros (abrir el archivo relacionado si está cerrado)</translation>
     </message>
     <message>
         <location line="+15"/>
         <source>File handling</source>
         <translation>Manejo de archivos</translation>
     </message>
     <message>
         <location line="+19"/>
@@ -4152,37 +4165,38 @@ Would you like to save the current short
     <message>
         <location line="+22"/>
         <source>Restore last directory of previous session</source>
         <translation>Restaurar último directorio de la sesión previa</translation>
     </message>
     <message>
         <location line="+32"/>
         <source>Initial file browser directory (only if not synchronized with initial working directory of Octave)</source>
-        <translation type="unfinished"></translation>
+        <translatorcomment>Traducción no muy buena</translatorcomment>
+        <translation type="unfinished">Directorio inicial del explorador de archivos (sólo si no está sincronizado con el directorio inicial de Octave)</translation>
     </message>
     <message>
         <location line="+36"/>
         <source>Extensions of files to be opened in the default text editor (separated by &quot;;&quot;):</source>
-        <translation>Extensiones de archivo para abrir en el editor de texto predeterminado (separado por &quot;;&quot;):</translation>
+        <translation>Extensiones de archivo para abrir en el editor de texto predeterminado (separadas por &quot;;&quot;):</translation>
     </message>
     <message>
         <location line="+39"/>
         <source>Workspace</source>
         <translation>Espacio de trabajo</translation>
     </message>
     <message>
         <location line="+35"/>
         <source>Colors for variable attributes</source>
         <translation>Colores para atributos de variables</translation>
     </message>
     <message>
         <location line="+107"/>
         <source>Use Command Window font</source>
-        <translation type="unfinished"></translation>
+        <translation>Usar la fuente de la ventana de comandos</translation>
     </message>
     <message>
         <location line="+196"/>
         <source>Import shortcut set</source>
         <translation>Importar conjunto de accesos directos</translation>
     </message>
     <message>
         <location line="+10"/>
@@ -4198,17 +4212,17 @@ Would you like to save the current short
         <location line="+3"/>
         <location line="+80"/>
         <source>Default</source>
         <translation>Valor predeterminado/por defecto</translation>
     </message>
     <message>
         <location line="-71"/>
         <source>Edit a shortcut by double-clicking in a cell</source>
-        <translation>Los accesos directos se pueden editar haciendo doble clic en la celda correspondiente</translation>
+        <translation>Editar el acceso directo haciendo doble clic en la celda correspondiente</translation>
     </message>
     <message>
         <location line="+76"/>
         <source>Actual</source>
         <translation>Actual</translation>
     </message>
     <message>
         <location line="+17"/>
@@ -4218,27 +4232,27 @@ Would you like to save the current short
     <message>
         <location line="+23"/>
         <source>Allow Octave to connect to the Octave web site to display current news and information</source>
         <translation>Permitir a Octave la conexión al sitio web para mostrar las últimas noticias e información</translation>
     </message>
     <message>
         <location line="+46"/>
         <source>Use proxy server</source>
-        <translation>Utilizar servidor &quot;proxy&quot;</translation>
+        <translation>Utilizar servidor proxy</translation>
     </message>
     <message>
         <location line="-470"/>
         <source>Variable Editor</source>
         <translation>Editor de variables</translation>
     </message>
     <message>
         <location line="+53"/>
         <source>Default row height</source>
-        <translation>Altura predeterminada de fila </translation>
+        <translation>Altura predeterminada de fila</translation>
     </message>
     <message>
         <location line="+17"/>
         <source>Default column width</source>
         <translation>Ancho predeterminado de columna</translation>
     </message>
     <message>
         <location line="+36"/>
@@ -4268,17 +4282,17 @@ Would you like to save the current short
     <message>
         <location line="+12"/>
         <source>Use alternating row colors</source>
         <translation>Usar colores de filas alternos</translation>
     </message>
     <message>
         <location line="+59"/>
         <source>Disable global shortcuts when Command Window has focus</source>
-        <translation type="unfinished"></translation>
+        <translation>Desactivar los accesos directos globales cuando el foco esté en la ventana de comandos</translation>
     </message>
     <message>
         <location line="+211"/>
         <source>HttpProxy</source>
         <translation>HttpProxy</translation>
     </message>
     <message>
         <location line="-2559"/>
@@ -4288,17 +4302,17 @@ Would you like to save the current short
     <message>
         <location line="+1929"/>
         <source>Synchronize Octave working directory with file browser</source>
         <translation>Sincronizar el directorio de trabajo de Octave con el explorador de archivos</translation>
     </message>
     <message>
         <location line="+385"/>
         <source>Shortcuts</source>
-        <translation>Accesos Directos</translation>
+        <translation>Accesos directos</translation>
     </message>
     <message>
         <location line="+31"/>
         <source>Select this option to prevent conflicts with readline shortcuts</source>
         <translation>Seleccione esta opción para evitar conflictos con los accesos directos de la línea de comandos</translation>
     </message>
     <message>
         <location line="+56"/>
@@ -4313,17 +4327,17 @@ Would you like to save the current short
     <message>
         <location line="+90"/>
         <source>Action</source>
         <translation>Acción</translation>
     </message>
     <message>
         <location line="+83"/>
         <source>Socks5Proxy</source>
-        <translation>&quot;Socks5Proxy&quot;</translation>
+        <translation>Socks5Proxy</translation>
     </message>
     <message>
         <location line="-16"/>
         <source>Hostname:</source>
         <translation>Nombre del host:</translation>
     </message>
     <message>
         <location line="+44"/>
